evalml/pipelines/time_series_classification_pipelines.py

```diff
@@ -78,7 +78,10 @@ def fit(self, X, y):
         X, y = self._convert_to_woodwork(X, y)
         self._encoder.fit(y)
         y = self._encode_targets(y)
-        X_t = self._compute_features_during_fit(X, y)
```

Comment:
I thought this was confusing because we called `self.input_feature_names = self.component_graph.input_feature_names` in `_compute_features_during_fit` and then again below! Moving just these two lines here helps remove the redundancy.